src/pages/home/report/ReportActionsView.tsx

```diff
@@ -246,43 +219,37 @@ function ReportActionsView(props) {
     }, [hasCachedActions]);
 
     // Comments have not loaded at all yet do nothing
-    if (!_.size(props.reportActions)) {
+    if (!reportActions.length) {
         return null;
     }
 
     return (
         <>
             <ReportActionsList
-                report={props.report}
+                report={report}
                 onLayout={recordTimeToMeasureItemLayout}
-                sortedReportActions={props.reportActions}
+                sortedReportActions={reportActions}
                 mostRecentIOUReportActionID={mostRecentIOUReportActionID}
                 loadOlderChats={loadOlderChats}
                 loadNewerChats={loadNewerChats}
-                isLoadingInitialReportActions={props.isLoadingInitialReportActions}
-                isLoadingOlderReportActions={props.isLoadingOlderReportActions}
-                isLoadingNewerReportActions={props.isLoadingNewerReportActions}
-                policy={props.policy}
+                isLoadingInitialReportActions={isLoadingInitialReportActions}
+                isLoadingOlderReportActions={isLoadingOlderReportActions}
+                isLoadingNewerReportActions={isLoadingNewerReportActions}
             />
             <PopoverReactionList ref={reactionListRef} />
         </>
     );
 }
 
-ReportActionsView.propTypes = propTypes;
-ReportActionsView.defaultProps = defaultProps;
 ReportActionsView.displayName = 'ReportActionsView';
+ReportActionsView.initMeasured = false;
```

Comment:
What this does?

Reply:
It lets TS know that ReportActionsView has static property initMeasured, which is used in this place:
https://github.com/Expensify/App/blob/174a0324028fcfcb1df7e82124a76ec9d805179e/src/pages/home/report/ReportActionsView.js#L239-L245